Sources/Service/WalletAvailabilityService.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import Networking
+import UIKit
+
+
+public final class WalletAvailabilityService {
```

Comment:
I'm wondering if this needs to be a class as it's quite lightweight and would only be needed here